### PR TITLE
Add auto disappear functionality

### LINKED ISSUE #3: Add auto disappear functionality
Add the ability to set a number of seconds before the notification self destructs.

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Added self-destruct functionality to the notification.

## QA
- Run `gulp build`
- Open the index.html file
- Check the cookie policy self-destructs after 3 seconds
- Update the duration on in the index.html and refresh
- Check the duration matches your change
- Remove the duration and check it never goes away

## Details
Fixes https://github.com/canonical-webteam/cookie-policy/issues/3

### CODE REVIEW

File: src/js/cookie-policy.js [hunk 1]
```diff
@@ -12,9 +12,20 @@ if (ubuntu.hasOwnProperty('cookiePolicy')) {
...
+      </a>.`
+  }
 
   return {
-    setup: function(content) {
+    setup: function(options) {
+      let content = options.content;
+      let duration = options.duration;
       let start = `
         <div class="p-notification--cookie-policy">
```
Comment (richmccartney) [inline src/js/cookie-policy.js]: As this is a popup and something that can be closed or hidden, I feel this should be using the ```<dialog>``` html element. Also add ```aria-labelledby``` or ```aria-describedby``` to give it more context and improve the accessibility.

File: src/js/cookie-policy.js [hunk 2]
```diff
@@ -12,9 +12,20 @@ if (ubuntu.hasOwnProperty('cookiePolicy')) {
...
           <p class="p-notification__content">`;
```
Comment (richmccartney) [inline src/js/cookie-policy.js]: The ```.js-close``` element should be a ```<button>``` element as we're not linking to a new page or anchor on the page.

File: index.html [hunk 3]
```diff
@@ -105,7 +105,11 @@
   </div>
   <script src="build/js/cookie-policy.js"></script>
   <script>
-    ubuntu.cookiePolicy.setup();
+    var options = {
+      'content': 'We use cookies to improve your experience. By your continued use of this site you accept such use.<br /> This notice will disappear by itself. To change your settings please <a href="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy#cookies">see our policy</a>.',
```
Comment (richmccartney) [inline index.html:109]: Does the ```<a>``` tag need to explain it is going to an external page?

**Review by richmccartney — CHANGES_REQUESTED** [review]

**Comment by anthonydillon** [comment]
@richmccartney thanks for the review. Nice pointers, feeling much more tab friendly now. Please, could you have another look?

**Review by richmccartney — APPROVED** [review]

**Comment by richmccartney** [comment]
🥇 a11y awesomeness, all LGTM and you can now merge this @anthonydillon

**Pull request merged**